tests/suites/test_suite_psa_crypto_storage_format.function

```diff
@@ -61,100 +61,6 @@ static int can_export(const psa_key_attributes_t *attributes)
     }
 }
 
-#if defined(MBEDTLS_TEST_LIBTESTDRIVER1)
```

Comment:
Of note, this workaround did not require listing in the ignore list for `do_analyze_driver_vs_reference` in `analyze_outcomes.py`, because it only affects what certain test cases do and not which test cases run.